examples/1_Introduction/03_preprocessing.ipynb

```diff
@@ -487,23 +537,25 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "b6e47d99",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# get acquisiiton geometry\n",
+    "# get acquisition geometry\n",
     "ag_centred = data_centred.geometry\n",
     "\n",
     "# FBP reconstruction\n",
     "fbp_recon_centred = FBP(ig, ag_centred, device='gpu')(data_centred)\n",
     "\n",
     "# visualize reconstruction results\n",
-    "show2D([fbp_recon_centred.subset(vertical=80), \\\n",
-    "        fbp_recon_centred.subset(vertical=100)], \\\n",
+    "show2D([fbp_recon_centred.get_slice(vertical=80), \\\n",
```

Comment:
this could (I think) be achieved with:
```
show2D(fbp_recon_centred, slice_list=[(vertical,80), (vertical,100)], cmap=cmap, num_cols=1, size=(10,10),origin='upper-left')"
```
or if the outer dimension is vertical 
```
show2D(fbp_recon_centred, slice_list=[80, 100], cmap=cmap, num_cols=1, size=(10,10),origin='upper-left')"
```